cirq-google/cirq_google/engine/engine.py

```diff
@@ -208,7 +209,7 @@ def __str__(self) -> str:
         return f'Engine(project_id={self.project_id!r})'
 
     @util.deprecated_gate_set_parameter
-    def run(
+    async def run_async(
```

Comment:
I think we're moving towards encouraging people to use EngineProcessor.run methods instead of `Engine` because `AbstractEngine` doesn't actually have any of these methods so they don't get mocked by the simulated versions.